centreon/www/front_src/src/Resources/Actions/index.test.tsx

```diff
@@ -613,14 +617,10 @@ describe(Actions, () => {
   });
 
   it('sends a check request when Resources are selected and the Check action is clicked', async () => {
```

Comment:
We might want to parametrize the test (uses test cases) to handle the forced check as well